manifests/init.pp

```diff
@@ -444,7 +444,12 @@
       true    => 'jpda start',
       default => 'start'
     }
-    $service_start_real = "${catalina_home_real}/bin/catalina.sh ${start_cmd}"
+    # catalina.sh in archive for 7,8,8.5 takes -security option to enable security manager
```

Comment:
Tomcat 6 as well (cf. https://tomcat.apache.org/tomcat-6.0-doc/security-manager-howto.html#Configuring_Tomcat_With_A_SecurityManager), therefore I would remove the "in archive for 7,8,8.5" part of that comment.